### ISSUE TITLE
Why do u use Spring cloud for microservice

### ISSUE THREAD

**Issue by HelloTo4x** [Cycas1994/nacos-spring-cloud#2]
Hi, i am reading your repository, i have a question, why do u use Spring cloud for microservice instead of Kubernetes, or AWS/GCP(cloud platform). Thank u so much

**Comment by Cycas1994** [comment]
Hi, my english is poor! nothing, i haven't got the Kubernetes, so i use Spring cloud.


------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "Cycas1994/nacos-spring-cloud"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2022年6月2日(星期四) 下午3:53
***@***.***&gt;;
***@***.***&gt;;
主题:&nbsp;[Cycas1994/nacos-spring-cloud] Why do u use Spring cloud for microservice (Issue #2)





 
Hi, i am reading your repository, i have a question, why do u use Spring cloud for microservice instead of Kubernetes, or AWS/GCP(cloud platform). Thank u so much

**Comment by HelloTo4x** [comment]
thanks, but have you ever met the problem about Spring cloud, can u share me about it

**Comment by Cycas1994** [comment]
when you met the problem about Spring cloud, u can share with me, let's talk about it


------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "Cycas1994/nacos-spring-cloud"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2022年6月2日(星期四) 下午4:46
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [Cycas1994/nacos-spring-cloud] Why do u use Spring cloud for microservice (Issue #2)





 
thanks, but have you ever met the problem about Spring cloud, can u share me about it

**Comment by HelloTo4x** [comment]
Yes. i am learning with it. but can you share me why do you use this : https://github.com/Cycas1994/nacos-spring-cloud/blob/master/elasticjob/src/main/java/com/cycas/elasticjob/listener/MyElasticJobListener.java , because Elastic search i know it usually use for Full text search, why don't you choose another database like Mongo....
